feat(home):  adjust homepage with new design

milhamm opened the pull request:
Closes #21 

- Adjust font family to use Open Sans
- Use existing `Button` component

cc @bandungpy/frontend

ridhof commented:
for < md screen size, something feels off to me. what do you think if we make the align to be center for < md screen size?
<img width="1448" alt="Screenshot 2023-07-31 at 20 14 21" src="https://github.com/bandungpy/pycon.id/assets/21301894/3a04bfb5-a74f-4c8c-b3b8-6493f3b94638">

mhaidarhanif commented:
Yes need to adjust a bit for the font sizing consistency

Large screen:

<img width="1000" alt="image" src="https://github.com/bandungpy/pycon.id/assets/235584/81ed2e13-84f2-441a-bf04-74060d664d75">

Medium screen:

<img width="1000" alt="image" src="https://github.com/bandungpy/pycon.id/assets/235584/ccfd977a-9965-49e3-8821-5c937bcdf6c8">

milhamm commented:
Will do, thanks @ridhof @mhaidarhanif

milhamm commented:
Updated, I've updated the font to Quicksand and adjusted the font consistency @mhaidarhanif @ridhof 

However @ridhof , when I tried to align the hero content to the center, it seems to be more off than it was before so I keep it as is. Any suggestion?

![image](https://github.com/bandungpy/pycon.id/assets/35242329/c54f1bf1-19a2-4bb8-b712-6003742d50fe)

mhaidarhanif commented:
Alright it's better

Let's keep it as is for now and merge this PR
